COM-1829:Store Not. Design SMS templates

Rameshdesaiise opened the pull request:
Added the sms template for store notification, and shipment Information.

Commit:
COM-1829:Store Not. Design SMS templates

Added the sms template for store notification, and shipment Information.

Commit:
Changes to labels to be a little more unique to mobile template and alph sorted the lang files

Commit:
COM-1829:Updated to model .shipmentNumber as api resturning the same

Commit:
COM-1829:Store Not. Design SMS templates

Updated shipment information template as per demo comments.
Move fulfilled items to top on shipment view.

added new table to display Items total,shipping,handling and total

updated total to subtotal.

Attaching the screen-shot for more details, please check and let me know if I need to do any updates in page.

michaelsadlonkibo commented on stylesheets/back-office.css:
```diff
@@ -194,3 +194,65 @@ table {
+    color: white !important;
+}
+
+ .shipment-Item-info{
+    font-weight: bold;
+    margin-left: 28px;
+    padding-top: 6px;
+}
+
+.shipment-Info-header{
+  background-color: #330065;
+    margin-bottom: 0!important;
+    padding: 8px 10px;
+    display: flex;
+    align-items: center;
+}
+
+.sidebar-kibo-logo{
+  height: 20px;
+    width: 107px;
+}
+
+.mobile-ShipmentInfo-table{
+  float: right;
+  background-color: #e5e5e5;
+  font-weight: normal;
+  padding: 8px 14px;
+  text-align: left;
+  white-space: nowrap;
+  width: 32%;
+  font-size: small;
+}
+
+.mobile-ShipmentInfo-table span{
+  float: right;
+}
```
Please fix the formatting (spaces, etc.) and make the classes all lowercase.
Example: `.mobile-shipment-info-table {`

michaelsadlonkibo commented on templates/mobile-notification/shipment-assignment.hypr:
```diff
@@ -0,0 +1,2 @@
+{% comment %} message length should not exceed 160 characters {% endcomment %}
+{{ labels.mobileLocationAssignedMessage|string_format(model.storeId, model.shipmentType, model.shipmenturl,model.fulfillerurl)|safe }}
```
Formatting please. Spaces after commas, etc.

michaelsadlonkibo commented on templates/mobile-notification/shipment-cancel.hypr:
```diff
@@ -0,0 +1,2 @@
+{% comment %} message length should not exceed 160 characters {% endcomment %}
+{{ labels.mobileShipmentCancelMessage|string_format(model.storeId, model.shipmenturl,model.fulfillerurl)|safe }}
```
Formatting please. Spaces after commas, etc.

michaelsadlonkibo commented on templates/back-office/mobile-notification-content.hypr:
```diff
@@ -0,0 +1,107 @@
+<div class="header shipment-Info-header">
```
Please format the HTML in this document. The tabbing is not consistent and some extra new lines can be removed.

michaelsadlonkibo commented on stylesheets/back-office.css:
```diff
@@ -194,3 +194,65 @@ table {
+.header .shipment-essentials .order-date .label {
+  font-weight: 600;
+}
+.shipment-Title {
```
Can we rename this `.shipment-title` (lowercase t)?

michaelsadlonkibo commented on labels/de-DE.json:
```diff
@@ -1,421 +1,431 @@
-﻿{
```
We shouldn't reformat/reorder the entire document at the same time we make changes. It's makes the diff really confusing. If clients want to pull the changes into custom templates it will be a pain.

I'd personally _only_ add the new fields for the language files.
If we want to reorganize the fields to be alphabetical we should do it _as a new commit_ separate from this PR.

michaelsadlonkibo commented on labels/en-US.json:
```diff
@@ -23,6 +23,7 @@
   "address": "Address",
```
Same as my other comment above. Shouldn't reorder this file.

Commit:
COM-1829:Store Not. Design SMS templates

1.Updated the shipment assigned message.
2.done the changes as per review comments.

Commit:
Update SMS text for cancel case
